employees/managers.py

```diff
@@ -3,20 +3,18 @@
 
 
 class EmployeeManager(BaseUserManager):
-    def create_user(self, username, password, first_name, last_name, employee_id, **extra_fields):
-        """
-        Create and save a User with the given username and password.
+    def create_user(self, username, password, **extra_fields):
+        """Create and save a User with the given username and password.
         """
         if not username:
             raise ValueError(_('The Username must be set'))
-        employee = self.model(username=username, first_name=first_name, last_name=last_name, employee_id=employee_id, **extra_fields)
+        employee = self.model(username=username, **extra_fields)
         employee.set_password(password)
         employee.save()
         return employee
 
     def create_superuser(self, username, password, **extra_fields):
-        """
-        Create and save a SuperUser with the given username and password.
+        """Create and save a SuperUser with the given username and password.
         """
```

Comment:
This doesn't conform to any of the styles allowed by [PEP257](https://www.python.org/dev/peps/pep-0257/).

```py
# Short docstrings should look like this:
"""Demonstrate a short docstring."""

# Longer docstrings should look like this:
"""
Demonstrate a multiline docstring.

There's a newline between the first line, which describes what the function or class does, 
and the rest of the body, which gives more details. We use proper spelling and punctuation
on all the lines.
"""